.github/workflows/docker.yml

```diff
@@ -208,3 +208,53 @@ jobs:
           from: ${{ secrets.NOTIFY_EMAIL_FROM }}
           to: ${{ secrets.NOTIFY_EMAIL_TO }}
           password: ${{ secrets.NOTIFY_EMAIL_PASSWORD }}
+
+  docker-ymax:
+    permissions:
+      # allow issuing OIDC tokens for this workflow run
+      id-token: write
```

Comment:
I don't have a strong understanding of OIDC in this context. I trust we'll get review from someone who does.

Reply:
copied from the others